### PR TITLE
Text Editors lesson: Update lists per layout style guide

### LINKED ISSUE #26437: Foundations/Prerequisites: Update lists per layout style guide
Lists should be updated in the following ways to align with our [layout style guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md):

- Any unordered list items that are created with an asterisk `*` are replaced with a hyphen `-` instead. For example, `* A list item` should be updated to `- A list item`
- The list item content is updated so it only has a single space before it. For example, `1.   A list item` has 3 spaces before the list item content ("A list item"), and should be updated to `1. A list item`
- Appropriate punctuation is added at the end of each list item in the "Lesson overview", "Assignment", "Knowledge check", and "Additional resources" sections
  - Follow the [Google style guide ](https://developers.google.com/style/lists#numbered-lettered-bulleted-lists) to determine when to add punctuation
- Any additional updates mentioned for specific lessons

### Guidance:

- Comment on this issue to let us know you'd like to get a lesson assigned to you. You can also ask for a specific one
- Please wait until you are assigned by a maintainer before starting to work on this issue to avoid multiple people working on the same lessons
- Make sure to read the [contributing guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md) before starting your first contribution
- Please add the following to your PR in the Issues section: `Related to #xxxxx` , since this issue is divided into sections, individual PR's should **NOT INCLUDE** the `closes` keyword. 

### Lessons to update

- [x] @scottwright-dev [Computer basics](https://www.theodinproject.com/lessons/foundations-computer-basics)
- [x] @rohan843  [How does the web work?](https://www.theodinproject.com/lessons/foundations-how-does-the-web-work)
- [x] @ht-l1  [Installation overview](https://www.theodinproject.com/lessons/foundations-installation-overview)
- [x] @Sushant9473  [Installations](https://www.theodinproject.com/lessons/foundations-installations)
- [x] @ScaxCodes  [Text editors](https://www.theodinproject.com/lessons/foundations-text-editors)
- [x] @VMadhuranga  [Command line basics](https://www.theodinproject.com/lessons/foundations-command-line-basics)
- [x] @matthewsanner  [Setting up git](https://www.theodinproject.com/lessons/foundations-setting-up-git)

### PR DESCRIPTION
**Pull request by ScaxCodes**
## Because

Update the lists syntax of the lessons to align with the TOP style guide.

## This PR

- Add punctuation at the end of unordered lists, following the google style guide.
- Change `*` to `-` syntax in unordered lists.

## Issue

Related to #26437

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

**Comment by ManonLef** [comment]
FYI it should be **Related to** #26437 and not **Closes** since there are multiple people working on the issue. Could you edit that in your description?

**Comment by ScaxCodes** [comment]
Thank you @ManonLef, I didn't notice that.

File: foundations/installations/text_editors.md [hunk 1]
```diff
@@ -25,22 +25,22 @@ Choose your Operating System:
 
 #### Step 1: Download VSCode
 
-   - Open your **Terminal**
-   - Run the following command to download the latest **VSCode** `.deb` package
+   - Open your **Terminal**.
+   - Run the following command to download the latest **VSCode** `.deb` package.
```
Comment (thatblindgeye) [inline foundations/installations/text_editors.md:29]:
```suggestion
- Open your **Terminal**.
- Run the following command to download the latest **VSCode** `.deb` package.
```

File: foundations/installations/text_editors.md [hunk 2]
```diff
@@ -25,22 +25,22 @@ Choose your Operating System:
...
 
 ~~~bash
 wget -O code-latest.deb 'https://code.visualstudio.com/sha/download?build=stable&os=linux-deb-x64'
 ~~~
 
 #### Step 2: Install VSCode
 
-   - Enter the following command in your terminal to install the **VSCode** `.deb` package
+   - Enter the following command in your terminal to install the **VSCode** `.deb` package.
```
Comment (thatblindgeye) [inline foundations/installations/text_editors.md:37]:
```suggestion
- Enter the following command in your terminal to install the **VSCode** `.deb` package.
```

File: foundations/installations/text_editors.md [hunk 3]
```diff
@@ -25,22 +25,22 @@ Choose your Operating System:
...
 
 ~~~bash
 sudo apt install ./code-latest.deb
 ~~~
 
-   - If prompted, enter your password
+   - If prompted, enter your password.
```
Comment (thatblindgeye) [inline foundations/installations/text_editors.md:43]:
```suggestion
- If prompted, enter your password.
```

File: foundations/installations/text_editors.md [hunk 4]
```diff
@@ -54,8 +54,8 @@ rm code-latest.deb
    
 You can start VSCode in two ways,
 
-   - Click **Visual Studio Code** from the Applications menu
-   - **Or**, use the `code` command from the terminal
+   - Click **Visual Studio Code** from the Applications menu.
+   - **Or**, use the `code` command from the terminal.
```
Comment (thatblindgeye) [inline foundations/installations/text_editors.md:58]:
```suggestion
- Click **Visual Studio Code** from the Applications menu.
- **Or**, use the `code` command from the terminal.
```

File: foundations/installations/text_editors.md [hunk 5]
```diff
@@ -72,20 +72,20 @@ code
 
 #### Step 2: Install VSCode
 
-   - Open the **Downloads** folder
-   - Double click the file **VSCode-darwin-universal.zip**
-   - Drag the **Visual Studio Code.app** icon to the **Applications** folder icon
+   - Open the **Downloads** folder.
+   - Double click the file **VSCode-darwin-universal.zip**.
+   - Drag the **Visual Studio Code.app** icon to the **Applications** folder icon.
```
Comment (thatblindgeye) [inline foundations/installations/text_editors.md:77]:
```suggestion
- Open the **Downloads** folder.
- Double click the file **VSCode-darwin-universal.zip**.
- Drag the **Visual Studio Code.app** icon to the **Applications** folder icon.
```

File: foundations/installations/text_editors.md [hunk 6]
```diff
@@ -72,20 +72,20 @@ code
...
 
 #### Step 3: Delete the installer file
 
-   - Open **Finder**
-   - Go to the **Downloads** folder
-   - Drag **VSCode-darwin-universal.zip** to the trash
+   - Open **Finder**.
+   - Go to the **Downloads** folder.
+   - Drag **VSCode-darwin-universal.zip** to the trash.
```
Comment (thatblindgeye) [inline foundations/installations/text_editors.md:83]:
```suggestion
- Open **Finder**.
- Go to the **Downloads** folder.
- Drag **VSCode-darwin-universal.zip** to the trash.
```

File: foundations/installations/text_editors.md [hunk 7]
```diff
@@ -72,20 +72,20 @@ code
...
 
 #### Step 4: Using VSCode
 
-   - Go to your **Applications** folder
-   - Double click **Visual Studio Code**
+   - Go to your **Applications** folder.
+   - Double click **Visual Studio Code**.
```
Comment (thatblindgeye) [inline foundations/installations/text_editors.md:88]:
```suggestion
- Go to your **Applications** folder.
- Double click **Visual Studio Code**.
```

**Review by thatblindgeye — CHANGES_REQUESTED** [review]
Some spacing changes noted below. Additionally can we remove the spacing from the Assignment numbered item?

**Comment by ScaxCodes** [comment]
So I did apply the suggestions from @thatblindgeye, now I want to add another commit as requested (remove spacing from the Assignment numbered item).

In this PR, there is this message:
> Add more commits by pushing to the [update_lists_layout](https://github.com/ScaxCodes/curriculum/tree/update_lists_layout) branch on [ScaxCodes/curriculum](https://github.com/ScaxCodes/curriculum).

But when I open my branch in VS Code, the suggestions mentioned above are not live in my local repo.
Is there a way to sync my local repo with the current state of the PR, or do I have to fix the spacing again manually in my code editor?

Whats the best practise?
@ManonLef can you help out?

**Comment by ScaxCodes** [comment]
Okay so I just played a bit around in the PR, and realized I can either edit the file right at Github in this PR and commit the change, or I can display and copy the code from the current state from this PR and paste it into my editor.

The consequences will vary a bit, I guess there will be a conflict when I commit the next change via my local repo, as this commit would try to change the suggestions from @thatblindgeye again, even its already changed in the PR.

I hope you can understand my logic, PR is just a new territory for me, and I would be happy if you can clarify this a bit.

I will commit via Github without my editor, I think its the best solution here.

**Comment by ManonLef** [comment]
@ScaxCodes you'd have to `pull` the branch to get the GH changes you've committed to your local branch as well :) But editing it on GH works fine too

**Comment by ScaxCodes** [comment]
@ManonLef I see, my branch gets updated during the PR, I just have to pull the current state via the CLI to update my local repo. I think I did not realize that my branch got updated, I thought the changes are isolated in this PR. Thank you!

**Review by thatblindgeye — APPROVED** [review]
Thanks for the updates! And glad you got things figured out, ➕➕  to @ManonLef for helping you out

**Pull request merged**